store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -147,6 +188,20 @@ type Request struct {
 	MvccGetByKey       *kvrpcpb.MvccGetByKeyRequest
 	MvccGetByStartTs   *kvrpcpb.MvccGetByStartTsRequest
 	SplitRegion        *kvrpcpb.SplitRegionRequest
+
+	DebugGet                    *debugpb.GetRequest
```

Comment:
What about remove these unused fields? This struct is already too large. Refer https://github.com/pingcap/tidb/issues/9641

Reply:
Okay.

Reply:
Have been removed.